ext/standaloneusers/Civi/Api4/Action/User/RequestPasswordResetEmail.php

```diff
@@ -42,16 +43,13 @@ public function _run(Result $result) {
     $userID = $user['id'] ?? 0;
 
     try {
-      // Allow flood control by extensions. (e.g. Moat).
-      $event = \Civi\Core\Event\GenericHookEvent::create([
-        'action'      => 'send_password_reset',
-        'identifiers' => ["user:$userID"],
-      ]);
-      \Civi::dispatcher()->dispatch('civi.flood.drip', $event);
```

Comment:
Bonus points - Also looks clean from `universe` POV. (No references to this event in my copy.)